dyno-contrib/src/main/java/com/netflix/dyno/contrib/EurekaHostsSupplier.java

```diff
@@ -127,7 +134,7 @@ public String getApplicationName() {
         return applicationName;
     }
 
-    public DiscoveryClient getDiscoveryClient() {
+    public EurekaClient getDiscoveryClient() {
```

Comment:
If someone assigns the return value to the DiscoveryClient type, they would get a compilation error.